sdv/cag/base.py

```diff
@@ -13,6 +13,27 @@ def __init__(self):
         self._fitted = False
         self._single_table = False
 
+    def _convert_data(self, data, metadata, copy=False):
```

Comment:
minor: can we call `convert_data_to_dictionary` or `dataframe_to_dict` or `dataframe_to_multi_table` or something along those lines?